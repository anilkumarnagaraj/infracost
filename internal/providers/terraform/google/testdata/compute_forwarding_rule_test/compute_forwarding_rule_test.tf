provider "google" {
  credentials = "{\"type\":\"service_account\"}"
  region      = "us-central1"
}

resource "google_compute_forwarding_rule" "default" {
  name = "website-forwarding-rule"
}
