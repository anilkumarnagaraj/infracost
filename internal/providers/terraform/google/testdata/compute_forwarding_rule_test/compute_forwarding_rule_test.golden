
 Name                                    Monthly Qty  Unit   Monthly Cost 
                                                                          
 google_compute_forwarding_rule.default                                   
 ├─ Forwarding rules                             730  hours         $7.30 
 └─ Ingress data                                 500  GB            $4.00 
                                                                          
 OVERALL TOTAL                                                     $11.30 
──────────────────────────────────
1 cloud resource was detected:
∙ 1 was estimated, it includes usage-based costs, see https://infracost.io/usage-file